### ISSUE TITLE
Entity yaw (mabye pitch) keeps going up

### ISSUE THREAD

**Issue by EmperialDev** [valence-rs/valence#121]
When you turn around your yaw (and properly also pitch) keeps goining up, so when you print out the player's yaw it is above 180 and if you turn the other way, you can get below -180.

Valence:
![Valence](https://user-images.githubusercontent.com/98261860/196517493-16e80ee9-1427-47f8-81f0-fdf163432ba1.png)
Vanilla:
![Vanilla](https://user-images.githubusercontent.com/98261860/196517495-a72e8fe9-6643-4eec-a507-ea21e9326698.png)

As you can see, this makes it so that the yaw (and mabye pitch) in the f3 doesn't match `client.yaw()` and `player.yaw()`.

Test plan
-
Print out the yaw and compare it to the f3 screen.

**Comment by jivvy** [comment]
@EmperialDev Is it important to make the yaw match what is shown on the client?

Not sure what kind of application you're using it for.

Although the yaw is not the same number, when completing revolutions around a circle with it, it should still point in the same direction.

Does this need to be fixed?

**Comment by EmperialDev** [comment]
I just discovered that the yaw kept going up, and thought that it was a bug, since when I printed out the value with a fabric mod, the yaw matched. But that is only if you log the yaw from the client, the server has the non capped yaw aswell on fabric.

But for what I use it for, it actually doesn't matter since the equation still works.

The only thing I can think of is the precision of the `f32` over time? Or what about if you want to teleport a player to another players yaw and pitch, than they will have the non capped yaw, is that okay to just send that, or does it need to be capped, I don't know? Another thing could be if someone else uses it, and just makes an if statment like this:
```rust 
if player.yaw > 90 && player.yaw < 180 {...}
```
But a simple comment would properly fix that.

So mabye not that important sry.
